src/replicate.cpp

```diff
@@ -228,6 +229,11 @@ void Replicate::command(int narg, char **arg)
   atom->extra_improper_per_atom = old->extra_improper_per_atom;
   atom->maxspecial = old->maxspecial;
 
+  if (old->labelmapflag) {
+    atom->add_label_map();
+    for (int i = 0; i<5; i++)  atom->lmap->merge_lmap(old->lmap, i);
```

Comment:
It's not wrong, but it seems a little strange to me to hard code a `5` here, rather than referring to a variable in the label_map class.

Reply:
@Bibobu thanks for this addition. looks good, but you could just write out each item in the list explicitly to avoid the technically dangerous iteration over enum. it would add several lines, but is probably most in line with how we designed the type labels framework. 

@sjplimp if this type of operation shows up again, we could add a better 'merge_all_lmaps' function